src/bootstrap/lib.rs

```diff
@@ -209,7 +209,7 @@ const EXTRA_CHECK_CFGS: &[(Option<Mode>, &'static str, Option<&[&'static str]>)]
     (Some(Mode::Std), "backtrace_in_libstd", None),
     /* Extra values not defined in the built-in targets yet, but used in std */
     (Some(Mode::Std), "target_env", Some(&["libnx"])),
-    (Some(Mode::Std), "target_os", Some(&["watchos"])),
+    (Some(Mode::Std), "target_os", Some(&["watchos", "trusty"])),
```

Comment:
Can you add `#[cfg(bootstrap)]` so that it will be removed in the next bootstrap (beta) bump.
```suggestion
    // #[cfg(bootstrap)] trusty
    (Some(Mode::Std), "target_os", Some(&["watchos", "trusty"])),
```

Reply:
It should be commented. The reason is because the `bootstrap` cfg key isn't available here, but we still want to be able to grep on it.

Reply:
Okay, thanks for clarifying!